ui/src/pages/Home/Profile/Profile.tsx

```diff
@@ -5,10 +5,10 @@ export function Profile() {
   return (
     <div className="profile">
       <div>
-      <img className="profile__image" src="bunnyW.jpg" alt="bunnie"/>
+        <img className="profile__image" src="bunnyW.jpg" alt="bunnie"/>
       </div>
       <div className="profile__text">
-      {text}{text}
+        {text}{text}
       </div>
     </div>
   )
```

Comment:
revert this change as it is not part of the ticket goal

Reply:
no

Reply:
I updated the ticket to include minor changes.

Reply:
AS BECAUSE of the ticket specifications now, this is an ok change